### ISSUE TITLE
Firefox keep loosing credentials

### ISSUE THREAD

**Issue by elr3m** [marius-wieschollek/passwords-webextension#186]
### System Information
- Extension Version: 2.1.2
- App Version: 2021.7.22
- Browser and Version: Firefox 90.0
- Client OS and Version: Windows 10 and MacOS 11

### Steps to reproduce
Firefox extension is set up with an app password, works for a few days and then within the extension "popup" I see the following message: Unable to connect to _servernickname_ server credentials rejected, please update the login data in the settings.
Happens repetitively on all Firefox extension after ~48 hours, regardless of the operating system, but it does not affect the Chrome extension.

### Expected result
No credentials error message

### Actual result
Message is surprising as the app password created for that extension has not been revoked, changed or modified

### 
[passwords_ext_logs.txt](https://github.com/marius-wieschollek/passwords-webextension/files/6833227/passwords_ext_logs.txt)
<details>
<summary>Browser log</summary>

</details>

Currently I have saved 1 of my app password and will try to reuse it on the next "extension disconnect" to see if the issue is with the app password or the extension. On Nextcloud, after the "extension disconnect", the app password shows as "a day ago" or more.

**Comment by ib54003** [comment]
Having the same issue since a few month... but seems to be a Windows only problem since i haven't experienced the issue on Fedora.

**Comment by elr3m** [comment]
I have a feeling it is related to Firefox synchronizing the settings of the extension.
Is the AppPassword used sync to Mozilla "cloud"?

**Comment by ib54003** [comment]
Yes i do use sync, i will disable it to check if that solves the problem.

**Comment by francoisPE** [comment]
Windows updated yesterday, and I loose my credential.
Impossible to be recognized by password add-on.  
It asked me to update 2021-11 to 2021-12... 
I try renew credential : no effect 
next, I remove addon and reinstall : now when I do connect from passlink everything goes well except that at the end, my credentials are not recorded...

**Comment by ib54003** [comment]
@francoisPE are u using Firefox sync?

**Comment by ib54003** [comment]
@francoisPE if yes try to disable it for the passwords app, that solved it for me !

**Comment by francoisPE** [comment]
Finally I solve it when I saw full message from another browser.
I needed to update my apps in nextcloud. I did it and all is back !
I take that time to tell you how amazing is this app and add-on : thanks a lot for that :-)
François

**Comment by schniepp** [comment]
Interesting. I have the same problem on Brave (chrome) in Linux!
Every time I restart the browser, the credentials to log on to my nextcloud instance are gone.

**Comment by ib54003** [comment]
> Interesting. I have the same problem on Brave (chrome) in Linux! Every time I restart the browser, the credentials to log on to my nextcloud instance are gone.
ur probably using firefox sync, disable it for the passwords app, that solved it for me !

**Comment by schniepp** [comment]
@ib54003 Actually, I don't have FF sync. This is a chrome-based browser, and NC passwords browser extension worked flawlessly for the longest time. I couldn't tell what I changed that could have affected it. I don't think I recently added any extensions to the browser. Somehow, it appears that the NC PW storage is suddenly non-persistent. 
The only other think that comes to mind is that I recently activated email as an additional 2-factor authorization mechanism. (I have had NC notifications for a long time.) However, since I am using an app pw for the extension, the 2FA should not affect authentication.
